src/main/java/org/apache/commons/lang3/math/NumberUtils.java

```diff
@@ -694,7 +703,7 @@ public static Number createNumber(final String str) {
         if (decPos > -1) { // there is a decimal point
             if (expPos > -1) { // there is an exponent
                 if (expPos < decPos || expPos > str.length()) { // prevents double exponent causing IOOBE
-                    throw new NumberFormatException(str + " is not a valid number.");
+                      throw new NumberFormatException(str + " is not a valid number.");
```

Comment:
You should try to avoid unnecessary changes like this one to whitespace - try to keep the diff to only changes that you need to make the typechecker pass.